apps/core-app/src/main/modules/box-tool/addon/preview/preview-registry.ts

```
@@ -1,65 +1,44 @@
-import type { PreviewAbilityResult } from '@talex-touch/utils'
-import type { PreviewAbility, PreviewAbilityContext } from './preview-ability'
+import type { PreviewAbility, PreviewSdk } from '@talex-touch/utils/core-box/preview'
+import { PreviewAbilityRegistry, createPreviewSdk } from '@talex-touch/utils/core-box/preview'
```

ESLint

[error] Expected "createPreviewSdk" to come before "PreviewAbilityRegistry".

(perfectionist/sort-named-imports)